common/addons/config-noise-function/src/main/java/com/dfsek/terra/addons/noise/samplers/noise/CellularSampler.java

```diff
@@ -540,6 +542,7 @@ public enum ReturnType {
         Distance2Mul,
         Distance2Div,
         NoiseLookup,
+        OffsetNoiseLookup,
```

Comment:
This might be more understandable as `LocalNoiseLookup`

Reply:
That sounds good, updated it